bin/gltf-pipeline.js

```diff
@@ -7,6 +7,8 @@ var addDefaults = require('../').addDefaults;
 var removeUnusedImages = require('../').removeUnusedImages;
 var removeUnusedSamplers = require('../').removeUnusedSamplers;
 var removeUnusedShaders = require('../').removeUnusedShaders;
+var removeUnusedTechniques = require('../').removeUnusedTechniques;
```

Comment:
As discussed offline, we'll merge #15 first, but I will still review this PR now.